.prettierignore

```diff
@@ -0,0 +1,6 @@
+# Ignore a bunch of files:
```

Comment:
This comment doesn't add any additional information that can't be inferred from the name of the file. Either improve the comment to explain _why_ we're ignoring, or just remove. I don't mind which.

Reply:
OK yep - removed the ignore file and linted some html and json